lib/Studio.js

```diff
@@ -512,7 +512,6 @@ module.exports = function(controller) {
                     json = JSON.parse(body);
                 } catch (e) {
                     console.log('Error in Botkit Studio Stats:', e);
-                    return reject('Invalid JSON');
```

Comment:
@anonrig the reject is still happening right outside the catch. we've seen weird behavior when calling callbacks inside try/catch blocks.